### ISSUE TITLE
Hardcoded "tabindex" in BootstrapTable.prototype.initHeader

### ISSUE THREAD

**Issue by igat-gh** [wenzhixin/bootstrap-table#2719]
Please, remove hardcoded attribute `tabindex="0"` for `th` tag or make it more flexible.
File: src/bootstrap-table.js
Line: 813

**Comment by wenzhixin** [comment]
Removed!

**Comment by pjabbott** [comment]
Removing this means you can no longer use keyboard navigation to select a header to sort the field, which makes the sort function inaccessible.  I work for a government agency and the applications I write cannot discriminate against people who are unable to use a mouse.

**Comment by djhvscf** [comment]
How can we improve this, @pjabbott?

we delete this [line](https://github.com/wenzhixin/bootstrap-table/commit/c88835421775200737219c920c485915e50a9672)
